README.md

```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
 the [Polkadot UI](https://substrate.dev/substrate-contracts-workshop/#/0/deploy-contract),
 as if your smart contract was written using ink!.
 
+## Roadmap
+
+Solang has a long way to go in order to achieve full compatibility for the blockchains we target.
+Here is a brief description of what we envision for the next versions.
+
+### V0.2
+
+| Milestone | Status |
+| --------- | ------ |
+| Specify values as "1 sol" and "1e9 lamports" | Not started |
+| Solana SPL tokens compatibility | In progess |
+
+### V0.3
+
+| Milestone | Status |
+| --------- | ------ |
+| Call Rust contracts from Solidity | Not started |
+| Parse and resolve inline assembly | In progress |
+
+### Contributions are welcome!
+
+These are the low-hanging fruits for first-time contributors:
```

Comment:
Ideally we should have a "first-time issues" label and bunch of issues with those labels.

Reply:
Should I remove this section?

Reply:
I think we should.

Having said that we should definitely create a bunch of "good first issue" for folks to contribute.